library/Garp/Db/Table/Rowset.php

```diff
@@ -49,37 +79,59 @@ public function flatten($column) {
     /**
      * Project a rowset using $fn
      *
-     * @param  callable $fn
+     * @param  callable $transform
      * @return Garp_Db_Table_Rowset
      */
```

Comment:
Due to the transforming to an array virtual field will get lost. I think it's good to mention that.